app/models/grda_warehouse/hud/organization.rb

```diff
@@ -12,13 +12,78 @@ class Organization < Base
       foreign_key: [:data_source_id, :organization_id], primary_key: [:data_source_id, :OrganizationID],
       inverse_of: :organization
     has_many :contacts, class_name: GrdaWarehouse::Contact::Organization.name, foreign_key: :entity_id
+    has_many :user_viewable_entities, as: :entity, class_name: 'GrdaWarehouse::Hud::UserViewableEntity'
```

Comment:
@eanders, I used the quoted class name rather than class.name here because I've learned that the latter causes difficulty for the Rails class loader and can produce weird bugs. I didn't change things generally in this file because I didn't want to create an enormous diff.